plugins/artifacts/artifacts.go

```diff
@@ -8,7 +8,7 @@ import (
 	"time"
 
 	schematypes "github.com/taskcluster/go-schematypes"
-	"github.com/taskcluster/taskcluster-client-go/tcclient"
+	"github.com/taskcluster/taskcluster-client-go"
```

Comment:
`tcclient` still I think... we previously had a really old version of taskcluster-client in `vendor/`...